runtime/compiler/control/DLLMain.cpp

```diff
@@ -357,7 +357,7 @@ IDATA J9VMDllMain(J9JavaVM* vm, IDATA stage, void * reserved)
             /* Platform detection seems difficult at this point.                              */
             {
 #ifdef TR_HOST_X86
-            static char *enableBatchClear = feGetEnv2("TR_EnableBatchClear", (void *)vm);
+            static char *disableBatchClear = feGetEnv2("TR_DisableBatchClear", (void *)vm);
```

Comment:
Why not pull this out of the `#ifdef` as part of the cleanup?  It is the same on both paths of the ifdef.